docs/client.rst

```diff
@@ -265,14 +265,15 @@ This implies:
 
 1. Flashes all output streams sending the data through the transport.
 2. If there is any reliable stream, it will perform the asociated reliable behaviour to ensure the communication.
-3. Listens messages from the agent and call the associated callback (a topic callback or a status callback).
+3. Listens messages from the agent and call the associated callback if exists (a topic callback or a status callback).
 
-The ``time`` suffix function version will perform these actions and will listen messages for a `time` duration.
+The ``time`` suffix function version will perform these actions and will listen messages for a ``time`` duration.
+Only when the time waiting for a message overcome the ``time`` duration, the function finished.
 The function will return ``true`` if the sent data have been confirmed, ``false`` otherwise.
 
 :session: Session structure previously initialized.
 :timeout: Time for waiting, in milliseconds.
```

Comment:
the parameter name is `time`, not `timeout`

Reply:
Done!